mtp_api/apps/performance/views.py

```diff
@@ -150,18 +156,18 @@ def get_context_data(self, **kwargs):
             )
 
         prisons = prisons.values()
-        order_by = 'nomis_id'
-        if (
-            'order_by' in self.request.GET and
-            self.request.GET['order_by'] in self.ordering_fields
-        ):
-            order_by = self.request.GET.get('order_by')
-
         prisons = sorted(
-            prisons, key=lambda p: p.get(order_by) or 0,
-            reverse=int(self.request.GET.get('desc', 0))
+            prisons, key=lambda p: p[form.cleaned_data['order_by']] or 0,
+            reverse=bool(form.cleaned_data['desc'])
         )
 
+        query_prefix = '&'.join(
+            '%s=%s' % (name, value)
+            for name, value in form.cleaned_data.items()
+            if name not in {'order_by', 'desc'}
+        )
+        context_data['form'] = form
+        context_data['order_by_prefix'] = query_prefix
```

Comment:
Why is this called `order_by_prefix`? Isn't it precisely all those things that _aren't_ related to `order_by`?